bindings/python/pymongocrypt/credentials.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2022-present MongoDB, Inc.
```

Comment:
I'm surprised these python changes are showing up here. Could you try rebasing?

Reply:
I'm also unsure why this happens. I've seen it across github several times. I'll try a rebase.

Reply:
Better now. I had accidentally cherry-picked a change when I intended to merge `master`.